common/src/main/java/com/wynntils/features/user/overlays/NpcDialogueOverlayFeature.java

```diff
@@ -195,8 +198,11 @@ private void renderDialogue(PoseStack poseStack, String currentDialogue) {
 
             // Render "To continue" message
             List<TextRenderTask> renderTaskList = new LinkedList<>();
-            TextRenderTask pressSneakMessage = new TextRenderTask("§cPress SNEAK to continue", renderSetting);
-            renderTaskList.add(pressSneakMessage);
+
+            if (!currentlyBlocking) {
```

Comment:
And here as well. `currentlyBlocking` is set to true if we get a blocking NPC dialogue, i.e. one which waits for sneak. So if, and only if, the dialog we are showing is blocking, then we should print this text message.